gcsstore.go

```diff
@@ -308,6 +308,10 @@ func (o *gcsFSObject) Open(accesslevel AccessLevel) (*os.File, error) {
 			}
 			defer rc.Close()
 
+			if _, err := cachedcopy.Seek(0, os.SEEK_SET); err != nil {
+				return nil, fmt.Errorf("error seeking to start of cachedcopy err=%v", err) //don't retry on local fs errors
+			}
+
 			_, err = io.Copy(cachedcopy, rc)
 			if err != nil {
 				errs = append(errs, fmt.Errorf("error coping bytes. err=%v", err))
```

Comment:
Is it that the downloaded file has some corruptions, or just somehow the reading of a section of the file was corrupted, I guess that would be the question?